tests/vmi_configuration_test.go

```diff
@@ -619,6 +623,59 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 			})
 		})
 
+		Context("with ACPI SLIC table", func() {
+			It("Should configure guest APCI SLIC with Secret file", func() {
+				const volumeSlicSecretName = "volume-slic-secret"
+				const secretWithSlicName = "secret-with-slic-data"
+				const hexData = "534c49432400000001494352415348204d450000000000008804000071656d7500000000"
```

Comment:
super nit: you can use a single const block

Reply:
Hi Alice, thanks for the quick review. I'll make it a const block in the next iteration. I'll probably move the binary data here as well and use that to compare (instead of `hexData`).